src/components/Auth/Login/Login.react.js

```diff
@@ -192,9 +193,11 @@ class Login extends Component {
       }
       case 'password': {
         const password = event.target.value.trim();
+        const passwordError = !(password.length >= 6);
         this.setState({
           password,
           passwordErrorMessage: !password ? 'Enter a valid password' : '',
+          passwordLength: passwordError,
```

Comment:
I think an extra passwordLength var was not needed. What you can do is, give the user an error saying that the password length should be at least 6. So we can make use of just the ```passwordErrorMessage``` var to handle that case. I hope you get my point.

Reply:
Yes, I got your point. 
But why to display the error if the password is lesser than 6 characters.
The button should automatically become active as 6 characters are entered. account.susi.ai works in the same manner. It doesn't display the error.

Reply:
Actually, we displayed an error in Signup so I thought the user should have a clear idea about why the button is not being activated. Up to you...